Zero Bounce iOS SDKTests/Zero_Bounce_iOS_SDKTests.swift

```diff
@@ -7,28 +7,527 @@
 //
 
 import XCTest
+import Mocker
 @testable import Zero_Bounce_iOS_SDK
 
 class Zero_Bounce_iOS_SDKTests: XCTestCase {
+    private let apiKey = "test-api-key"
 
     override func setUp() {
-        // Put setup code here. This method is called before the invocation of each test method in the class.
+        ZeroBounceSDK.shared.initialize(apiKey: apiKey)
     }
 
     override func tearDown() {
         // Put teardown code here. This method is called after the invocation of each test method in the class.
     }
 
-    func testExample() {
-        // This is an example of a functional test case.
-        // Use XCTAssert and related functions to verify your tests produce the correct results.
+    func testGetCredits() {
+        let apiEndpoint = URL(
+            string: "\(ZeroBounceSDK.shared.apiBaseUrl)/getcredits?api_key=\(ZeroBounceSDK.shared.apiKey!)"
+        )!
+        let credits = "3000"
+        let creditsResponse = ZBCreditsResponse(credits: credits)
+        let mockedData = try! JSONEncoder().encode(creditsResponse)
+        let mock = Mock(url: apiEndpoint, dataType: .json, statusCode: 200, data: [.get: mockedData])
+        mock.register()
+
+        let requestExpectation = expectation(description: "Request should finish")
+        ZeroBounceSDK.shared.getCredits() { result in
+            defer { requestExpectation.fulfill() }
+            switch result {
+            case .Success(let response):
+                NSLog("getCredits success response=\(response)")
+                XCTAssertEqual(response.credits, credits)
+            case .Failure(let error):
+                NSLog("getCredits failure error=\(String(describing: error))")
+                XCTAssertTrue(false)
+            }
+        }
+
+        wait(for: [requestExpectation], timeout: 60)
+    }
+    
+    func testGetCreditsStatusError() {
+        let apiEndpoint = URL(
+            string: "\(ZeroBounceSDK.shared.apiBaseUrl)/getcredits?api_key=\(ZeroBounceSDK.shared.apiKey!)"
+        )!
+        let credits = "3000"
+        let creditsResponse = ZBCreditsResponse(credits: credits)
+        let mockedData = try! JSONEncoder().encode(creditsResponse)
```

Comment:
Not necessarily the case here, as you know what the encodable object contains, but usually it's preferable to 

`do {
    let mockedData = try JSONEncoder().encode(creditsResponse)
} catch { ... }`

OR, if you don't care if it fails, just do

`let mockedData = try? JSONEncoder().encode(creditsResponse)`

So avoid using try!, either encompass a regular try in a do-catch, or use try?